README.md

```diff
@@ -104,29 +173,17 @@ mvn clean validate -P checkstyle
 mvn clean compile -P spotbugs
 ```
 
-### Contributing to Hazelcast
-
-We encourage Pull Requests and process them promptly.
-
-To contribute:
-
-* see [Developing with Git](https://hazelcast.atlassian.net/wiki/display/COM/Developing+with+Git) for our Git process
-* complete the [Hazelcast Contributor Agreement](https://hazelcast.atlassian.net/wiki/display/COM/Hazelcast+Contributor+Agreement)
-
-For an enhancement or larger feature, create a GitHub issue first to discuss.
-
-
-### Mail Group
+## Mail Group
```

Comment:
I would rename this section to something like "Get help" and mention the Gitter channel as well as Stackoverflow (maybe not necessary)

Reply:
And I would move the sections about contributing and this get help close together, because it is usually related.